src/PropertyInfo/TypeInfoParsers/NativeTypeInfoParser.php

```diff
@@ -19,11 +17,8 @@
  *
  * @author Mihai Stancu <stancu.t.mihai@gmail.com>
  */
-abstract class NativeTypeInfoParser implements TypeInfoParserInterface
+trait NativeTypeInfoParser
```

Comment:
I see the `symfony` branch is a big refactoring. I'll just port the `Hack` related functionality into that new structure.

I'll close this PR as unnecessary.